Contributing/How-to-Start-a-New-Project.md

```diff
@@ -0,0 +1,28 @@
+## Start a new Project
+
+Keploy is increasingly looking for new developers to join us and help improve our processes. We are also looking to collaborate on the development of new projects that serve our plan and mission. Please have a look at the following steps to start a new project:
+
+1.  Avoid Duplication:
+
+    -   Before starting a new project, review whether a project based on your idea already exists:
+        -   All projects are listed in the **Project** tab of the keploy root directory.
+        -   Please also review the existing repositories, as not all may be listed under the "Project" tab.
+    -   If your project idea already exists, please join forces with those already active. Else (if your project does not exist) proceed with the next step.    
+
+2.  Build on what already exists:
+
+    -   If you can build on code that is already available, coordinate your project with the maintainers of the relevant repository by [requesting a new feature]().
+    -   If there is no existing code base to start from, [create a repository](How-To-Create-a-Repository.md) even if you are not part of the keploy organisation (yet).
+
+3.  Create a project board to track and update your work:
+
+    -   If you are a part of the keploy organisation use the project list under `.github`. 
+    -   If you are not (yet) part of the keploy organisation, use the projects list in your repository. You can notify the wider keploy community by opening an issue under `.github` informing that you have started a new project.
+
+4.  Invite your team:
+
+    -   [Invite your team](https://help.github.com/en/articles/adding-organization-members-to-a-team) to join the project even [if they are not part of the keploy organisation](https://help.github.com/en/articles/adding-outside-collaborators-to-repositories-in-your-organization)
+    -   [Nest your team](https://help.github.com/en/articles/moving-a-team-in-your-organizations-hierarchy) into the right parent team, which means the repository team for new projects or the keploy team for new repositories   
+    -   Assign roles for Developers, Engineers, Editors, Maintainers (for code). Try to have at least 2 persons for each role to avoid overloading your team. 
+
+5.  Follow the [regular project implementation cycle]()
```

Comment:
Why is there an empty link?

Reply:
Because many folders and many things are not done